assets/js/components/HealthSummary/HomeHealthSummary.test.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { screen, fireEvent } from '@testing-library/react';
+import 'intersection-observer';
+import '@testing-library/jest-dom';
+import { keysToCamel } from '@lib/serialization';
+
+import { HomeHealthSummary } from './HomeHealthSummary';
+import { renderWithRouter, withState } from '../../lib/test-utils';
+import {
+  setHealthSummary,
+  stopHealthSummaryLoading,
+} from '../../state/healthSummary';
+import { act } from 'react-dom/test-utils';
+
+const homeHealthSummaryPayload = [
+  {
+    clusterId: '7965f822-0254-5858-abca-f6e8b4c27714',
```

Comment:
Can we use the new factory libraries to create this test data?

Reply:
you're right! done right know!